### ISSUE TITLE
Understand proposed changes in WCAG 2.2 draft

### ISSUE THREAD

**Issue by 36degrees** [alphagov/govuk-design-system#1321]
## What

Understand the new and updated success criteria in the working draft of WCAG 2.2, applying them to the components and patterns in the Design System, considering where potential failures may exist and what sort of changes would be needed to correct them.

Play back what we learn to the rest of the team (and any other interested parties).

## Why

WCAG 2.2 is current a 'First Public Working Draft'. The [expected schedule as of February was](https://www.w3.org/blog/2020/02/wcag-22-in-development/):

> We plan to publish a final draft (W3C Candidate Recommendation) in June 2020, and hope to publish the final version of WCAG 2.2 as a “W3C Recommendation” web standard later in 2020.

The final draft has not yet been published, but [it looks as though the Accessibility Guidelines Working Group are still aiming to publish by the end of the year](https://twitter.com/alastc/status/1286022298244395015).

Our current understanding is that The Public Sector Bodies (Websites and Mobile Applications) (No. 2) Accessibility Regulations 2018 will continue to reference WCAG 2.1 until EN 301 549 is updated. However, we should still be aiming to conform to the new and updated success criteria in WCAG 2.2 because:

- we have a responsibility to ensure our products can be used to build services that are as accessible as we can make them, and not just to ensure we are meeting the 'minimum' defined in the legislation
- we expect that third party auditors may start assessing against WCAG 2.2 before a service is legally required to conform to it
- by understanding WCAG 2.2 whilst it is still in a draft phase, we may be able to feed back where there are gaps or ambiguity in the guidance, which will help us in the future when services are being audited
- by 'getting ahead of the legislation' and correcting any failures now, we reduce the likelihood that a service team will need to update through a breaking change under pressure

## New Criteria

3.3.7 	Accessible Authentication (A) – Vanita
2.5.7 	Dragging (AA) – Ollie
3.2.6 	Findable Help (A) – Mark
2.4.13 	Fixed Reference Points (A) – ?
2.4.11 	Focus Appearance (Minimum) (AA) – Ollie
2.4.12 	Focus Appearance (Enhanced) (AAA) – Ollie
3.2.7 	Hidden Controls (AA) – Vanita
2.5.8 	Pointer Target Spacing (AA) – Hanna
3.3.8 	Redundant Entry (A) – Hanna

## Done when…

- [x] Read and understand the changes in the current editor's draft of WCAG 2.2
- [x] Put together a high level summary of the changes and what they mean for the components and patterns in the Design System
- [x] Identify any unclear or ambiguous success criteria that we may want to feed back on
- [x] Play back to the team

**Comment by 36degrees** [comment]
Have made a start on this here:
https://docs.google.com/presentation/d/1ev-8bxwMmQS59WVJd6a7xv7ixEcqY_XTCr6j9oJ78kE/edit#slide=id.g8f34273a21_0_288

At the minute I am looking at the new 2.4.11 Focus Visible (enhanced) success criteria.

Hanna is going to look at SC 2.5.8 Pointer Target Spacing and SC 3.3.8 Redundant Entry.

Suggest we aim to have something we can play back to the rest of the team in the Show & Tell on 18 August?

**Comment by 36degrees** [comment]
I can think of 4 things that I would like us to feed back about focus states:

- The maths for calculating the border area does not seem to add up…
- Ask for clarification on how the focus indication area should be calculated when an inline control wraps over multiple lines (for example a link)
- Provide an example of short links as an area where the visible indicators 'fail' (given "The working group is interested in feedback about the minimum area metric, and if there are unusual scenarios where visible indicators are caught by the wording.")
- Provide an example of the accordion as a component that has two 'visible' boundaries for the same click area, and ask for clarification about how the minimum focus indication area should be calculated.

**Comment by 36degrees** [comment]
Now raised:

> * The maths for calculating the border area does not seem to add up…

https://github.com/w3c/wcag/issues/1324

> * Ask for clarification on how the focus indication area should be calculated when an inline control wraps over multiple lines (for example a link)

https://github.com/w3c/wcag/issues/1328

> * Provide an example of short links as an area where the visible indicators 'fail' (given "The working group is interested in feedback about the minimum area metric, and if there are unusual scenarios where visible indicators are caught by the wording.")

https://github.com/w3c/wcag/issues/1329

> * Provide an example of the accordion as a component that has two 'visible' boundaries for the same click area, and ask for clarification about how the minimum focus indication area should be calculated.

https://github.com/w3c/wcag/issues/1075#issuecomment-677676912

**Comment by 36degrees** [comment]
@m-green has also mentioned feeding back on 3.2.6 Findable Help, but is unlikely to have time to do so this sprint.

Leaving this open for now – hopefully we can pick it up next sprint or before the 18 September deadline for feedback (https://www.w3.org/blog/2020/08/wcag22-wide-review/).

**Comment by m-green** [comment]
I've now raised the issue with WCAG implicitly recommended FAQs and 'How do I...' pages.

https://github.com/w3c/wcag/issues/1380

**Comment by 36degrees** [comment]
Is there anything else we wanted to feed back on? Can we close this issue?

**Comment by m-green** [comment]
Nothing more from me.
